recipe/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "st-theme" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/st-theme-{{ version }}.tar.gz
+  sha256: ca97aece1a48ded6e83fd742c27cb0851e1bce2100ab4b6c37c7b6e003b65b42
+
+build:
+  entry_points:
+    - st-theme = streamlit_theme:print_version
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [s390x or py<38]
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - streamlit >=1.33
+
+test:
+  imports:
+    - streamlit_theme
+  commands:
+    - pip check
+    - st-theme --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/gabrieltempass/streamlit-theme
+  summary: A component that returns the active theme of the Streamlit app.
+  description: |
+    The function immediately returns the active theme, 
+    when it is called. If the user manually changes the theme, 
+    after the web app is already running, it updates the returned value.
+  license: MIT
+  license_file: LICENSE
+  license_family: MIT
+  dev_url: https://github.com/gabrieltempass/streamlit-theme/
+  doc_url: https://github.com/gabrieltempass/streamlit-theme/blob/main/README.md
```

Comment:
`setup.py` is saying use the GH repo: https://github.com/gabrieltempass/streamlit-theme/blob/1.2.3/setup.py which would present you with the README as a side-effect.  🤷